mods/sp/rules/vehicles.yaml

```diff
@@ -1050,7 +1050,7 @@ TICKHOLOGRAM:
 	RenderSprites:
 		Image: ttnk
 	FirepowerMultiplier@Hologram:
-		Modifier: 20
+		Modifier: 0
```

Comment:
These need to have a modifier of atleast 1 or 2 to make sure the holograms apply take cover effects and damage notifications.

Reply:
I drop the modifier and make a new weapon for them, which also fix the break shield bug for holograms.